rules_building_block/discovery_generic_account_groups.toml

```diff
@@ -52,20 +58,23 @@ process where host.os.type == "windows" and event.type == "start" and
       process.args : "set"
     )
   )
-) and not user.id : "S-1-5-18"
+) and not process.parent.args: "C:\\Program Files (x86)\\Microsoft Intune Management Extension\\Content\\DetectionScripts\\*.ps1"
```

Comment:
I like the BBR rule type changes to new terms, but I am not sure that we should chase tuning BBRs for things like this, since they are meant to be more contextual, lower maintenance rules.

Reply:
Yeah I only did it for the ones that were just too noisy to be part of the UEBA pack IMO. I agree that BBR makes sense to not be new_terms, and to detect small parts of behavior, so being noisy is less of an issue. But with these changes I feel confident that I can add them to the Discovery pack without risk of noise. 

100% agreed that, for default DRs, a bit of noise in BBR is fine.